interface/HHAnalyzer.h

```diff
@@ -105,20 +109,21 @@ class HHAnalyzer: public Framework::Analyzer {
         void fillTriggerEfficiencies(const Lepton & lep1, const Lepton & lep2, Dilepton & dilep);
 
         // global event stuff (selected objects multiplicity)
-        BRANCH(nJets, unsigned int);
+        ONLY_NOMINAL_BRANCH(nJets, unsigned int);
         BRANCH(nJetsL, unsigned int);
```

Comment:
Indeed, but with your changes it's now equivalent to `nJets`, so I can switch to that, and then we do not need to store `nJetsL`, `nBJetsL`, `nBJetsT`, `nMuonsT`, `nElectronsM`, since you're keeping only the objects passing our IDs?